http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -93,6 +93,7 @@ public VerifiedTokens parseAndVerifyToken(final String idToken, final String acc
             JwtClaims jwtClaims = new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(accessToken);
             return new VerifiedTokens(new IDToken(idJwtClaims), new AccessToken(jwtClaims));
         } catch (InvalidJwtException e) {
+            log.debugf("Problem parsing ID token: " + idToken, e);
```

Comment:
Thanks for taking this one @ivassile! Just to be consistent with the other logging messages for OIDC, I think it would be good to use `tracef` here.